README.md

```diff
@@ -4,73 +4,88 @@ This repository contains reproducible benchmarks to evaluate the instrumentation
 ```
 
-## How It Works
-
-The `sentry-sdk-benchmark` command automates the following steps:
+We provide a [collection of prepackaged web apps](platform/) implemented in different programming languages and using different frameworks.
 
-1. Start `baseline` app:
+For each original app in a "baseline" directory, there is a slightly modified app in a corresponding "instrumented" directory which is a copy of the original app with minimal changes to add Sentry instrumentation (error and performance monitoring).
 
-    ```shell
-    (cd platform/python/django/baseline && docker compose -p django-baseline up -d --build)
-    ```
+The benchmark runner (`sentry-sdk-benchmark` tool) takes one or more apps as input and creates, serially, Docker Compose projects to test and gather data to compare baseline and instrumented apps. The output is presented as an HTML report.
 
-2. Run load generator as a warm-up step:
+Each Docker Compose project is responsible for spinning up a target app, a database server, and auxiliary tools to generate load and measure latencies and resource consumption. Those components exist as a handful of Docker containers:
 
-    ```shell
-    <<<'GET http://localhost:8080/update?queries=10' vegeta attack -duration 10s -rate 500/1s | vegeta report
-    ```
+```
+   ┌────────────────────────────────────────────────────────────────────────┐
+   │                                                                        │
+   │ Container Metrics Collector                                            │
+   │         (cAdvisor)                                                     │
+   │                                                                        │
+   └────────────────────────────────────────────────────────────────────────┘
+   ┌────────┐ ┌──────────────┐ ┌───────────────────────┐ ┌──────────────────┐
+   │        │ │              │ │                       │ │                  │
+   │ Target │ │   Database   │ │    Load Generator,    │ │   Mock Sentry    │
+   │  App   │ │ (PostgreSQL) │ │    Data Collector     │ │ Ingestion Server │
+   │        │ │              │ │ and Test Orchestrator │ │   (fakerelay)    │
+   │        │ │              │ │      (loadgen)        │ │                  │
+   │        │ │              │ │                       │ │                  │
+   └────────┘ └──────────────┘ └───────────────────────┘ └──────────────────┘
+                                                          * only for
+                                                            instrumented apps
+```
 
-    You may repeat the command above a couple of times, observing that after the first run the maximum latency reported should drop down.
+Every app under test is supposed to interact with the database in response to requests from the load generator as described in the [Database Updates test](https://github.com/TechEmpower/FrameworkBenchmarks/wiki/Project-Information-Framework-Tests-Overview#database-updates).
```

Comment:
```suggestion
Every app under test is supposed to interact with a postgres database in response to requests from the load generator as described in the [Database Updates test](https://github.com/TechEmpower/FrameworkBenchmarks/wiki/Project-Information-Framework-Tests-Overview#database-updates).
```

Reply:
👍 

Will use "PostgreSQL" throughout for consistency.

```suggestion
Every app under test is supposed to interact with a PostgreSQL database in response to requests from the load generator as described in the [Database Updates test](https://github.com/TechEmpower/FrameworkBenchmarks/wiki/Project-Information-Framework-Tests-Overview#database-updates).
```